service/lib/agama/manager.rb

```diff
@@ -58,18 +62,22 @@ def initialize(config, logger)
       @logger = logger
       @installation_phase = InstallationPhase.new
       @service_status_recorder = ServiceStatusRecorder.new
+      @service_status = DBus::ServiceStatus.new.busy
```

Comment:
This looks like the manager will only rarely be _idle_ and [`safe_run`](https://github.com/openSUSE/agama/blob/40e46fa183077be4115de6491650c7957f92f121/service/lib/agama/dbus/manager.rb#L146) will fail more often?

Reply:
I would not say rarely, but this ensures that it waits for startup phase, not?

Reply:
Yes, that's the point. It starts as busy, but it is set as idle as soon as the startup_phase finishes.

Reply:
Ok then